components/brave_vpn/resources/panel/components/error-panel/style.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
+
+export const Box = styled.div`
+  width: 100%;
+  height: 100%;
+  max-height: 450px;
+  background: ${(p) => p.theme.color.panelBackground};
+  position: relative;
+  overflow: hidden;
+`
+
+export const PanelContent = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 48px 24px 16px 24px;
+  position: relative;
+  z-index: 2;
+`
+
+export const ReasonTitle = styled.h1`
+  color: ${(p) => p.theme.color.text01};
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 18px;
+  font-weight: 600;
+  letter-spacing: 0.02em;
+  line-height: 2.6;
+  margin: 0;
+  text-align: center;
+`
+
+export const ReasonDesc = styled.p`
+  color: ${(p) => p.theme.color.text01};
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0.02em;
+  margin: 0;
+  text-align: center;
+  padding-bottom: 24px;
+`
+
+export const IconBox = styled.div`
+  width: 62px;
+  height: 62px;
+
+  svg>path {
+    fill: ${(p) => p.theme.color.text03};
+  }
+`
+
+export const ActionArea = styled.div`
+  width: 100%;
+
+  button {
```

Comment:
Perhaps prefer `${Button}` here?

Reply:
I'm not sure if i understand why we need this. Assuming Button is already a definition and it should have base properties of button? brave-ui's button component doesn't have an option to make the width grow, well it does have one but not the right size then.